modules/custom-tuning-example.adoc

```diff
@@ -43,3 +42,45 @@ Custom profile writers are strongly encouraged to include the default TuneD
 daemon profiles shipped within the default Tuned CR. The example above uses the
 default `openshift-control-plane` profile to accomplish this.
 ====
+
+*Using built-in TuneD profiles*
+
+Given the successful rollout of the NTO-managed DaemonSet,
+the TuneD operands all manage the same version of the
+link:https://github.com/redhat-performance/tuned/[TuneD daemon].  To list
+the built-in TuneD profiles supported by the daemon, query any TuneD pod
+in the following way:
+
+[source,yaml]
+----
+$ oc exec $tuned_pod -n openshift-cluster-node-tuning-operator -- find /usr/lib/tuned/ -name tuned.conf -printf '%h\n' | sed 's|^.*/||'
+----
+
+You can use the profile names retrieved by this in your custom tuning specification.
+
+.Example: using built-in hpc-compute TuneD profile
+[source,yaml]
+----
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: openshift-node-hpc-compute
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=Custom OpenShift node profile for HPC compute workloads
+      include=openshift-node,hpc-compute
+    name: openshift-node-hpc-compute
+
+  recommend:
+  - match:
+    - label: tuned.openshift.io/openshift-node-hpc-compute
+    priority: 20
+    profile: openshift-node-hpc-compute
+----
+
+Note the example above includes the default `openshift-node` TuneD daemon profiles shipped within the default Tuned CR. This is to take advantage of the OpenShift specific tuning for compute nodes.
```

Comment:
s/Note the example/The example

s/This is to take advantage of the OpenShift specific/The default profiles use OpenShift-specific tuning

Reply:
The second change would imply that the default profiles provide OpenShift-specific tuning to compute nodes only. This is not true. The default profiles also provides tuning for the control plane. Would changing the entire sentence to:

```
In addition to the built-in `hpc-compute` profile, the example above includes
the `openshift-node` TuneD daemon profile shipped within the default
Tuned CR to use OpenShift-specific tuning for compute nodes.
```

help?

Reply:
@jmencak Your updates look good to me. Thank you!